packages/calcite-components/src/components/combobox-item-group/combobox-item-group.tsx

```diff
@@ -50,6 +50,24 @@ export class ComboboxItemGroup {
 
   scale: Scale = "m";
 
+  // --------------------------------------------------------------------------
+  //
+  //  Private Methods
+  //
+  // --------------------------------------------------------------------------
+
+  private isPaddingPresent(): boolean {
```

Comment:
nitpick: can we come up with a better name for this private method? Something that describes what the function is testing for.